lib/Setup/utilitybrandedtoken.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @typedef {Object} UtilityBrandedTokenSetupConfig
+ *
+ * @property {string} valueToken Address of the branded token contract on
+ *                               origin.
+ * @property {string} symbol Symbol for branded token contract.
+ * @property {string} name Name for branded token contract.
+ * @property {string} decimal Decimal for branded token contract.
+ * @property {string} organization The address of the organization contract
+ *                                 on auxiliary chain.
+ */
+
+'use strict';
+
+const { ContractInteract, Setup } = require('@openstfoundation/mosaic.js');
+const UtilityBrandedToken = require('../ContractInteract/UtilityBrandedToken');
+
+/**
+ * A single function to setup a utility branded token. This function deploy
+ * auxiliary organization, EIP2Gateway, EIP20CoGateway and Utility branded
+ * token.
+ *
+ * @param {Web3} originWeb3 Web3 instance pointing to origin chain.
+ * @param {Web3} auxiliaryWeb3 Web3 instance pointing to auxiliary chain.
+ * @param {OrganizationSetupConfig} auxiliaryOrganizationConfig Configuration to
+ *                                                              deploy auxiliary
+ *                                                              organization.
+ * @param {Object} auxiliaryOrganizationTxOptions Transaction options to deploy
+ *                                                auxiliary organization.
+ * @param {UtilityBrandedTokenSetupConfig} auxiliaryUBTConfig Configuration to
+ *                                                            deploy utility
+ *                                                            Branded token.
+ * @param {Object} auxiliaryTxUBTOptions Transaction options to deploy
+ *                                       utility Branded token.
+ * @param {GatewaysOriginSetupConfig} originGatewayConfig Configuration to
+ *                                                        deploy origin gateway.
+ * @param {GatewaysAuxiliarySetupConfig} auxiliaryGatewayConfig Configuration to
+ *                                                              deploy auxiliary
+ *                                                              co-gateway.
+ * @param {Object} originGatewayTxOptions Transaction options to deploy
+ *                                        origin gateway.
+ * @param {Object} auxiliaryCoGatewayTxOptions Transaction options to deploy
+ *                                             auxiliary co-gateway.
+ * @param {Object }auxiliaryUBTSetCoGatewayTxOptions Transaction options to set
+ *                                                   co-gateway on utility
+ *                                                   branded token. From
+ *                                                   address in transaction
+ *                                                   must be organization
+ *                                                   master key.
+ *
+ * @return {Promise<{
```

Comment:
Github complains. According to this, you should define it separately and return it: https://stackoverflow.com/questions/28763257/jsdoc-return-object-structure